gradle-baseline-java-config/resources/findbugs/excludeFilter.xml

```diff
@@ -18,4 +18,16 @@
             <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"/> <!-- @Rule ivars can be public and unused -->
         </Or>
     </Match>
+
+    <!-- Ignore the following bug patterns in Immutables generated code -->
+    <!-- (i.e., classes beginning with 'Immutable' and inner classes of same) -->
+    <Match>
+        <Class name="~.*Immutable.*(\$.*)?" />
```

Comment:
is there a chance that people will use ImmutableFoo when it's not generated? I wonder if we should move to a prefix convention like "Immutables_" instead. Or maybe this block doesn't actually belong in baseline because not everyone uses Immutables.